starknet_py/transactions/declare.py

```diff
@@ -24,10 +24,9 @@ def make_declare_tx(
     :param cairo_path: a ``list`` of paths used by starknet_compile to resolve dependencies within contracts
     :return: A "Declare" transaction object
     """
-    if isinstance(compiled_contract, str):
-        compiled_contract = create_compiled_contract(
-            compilation_source, compiled_contract, cairo_path
-        )
+    compiled_contract = create_compiled_contract(
+        compilation_source, compiled_contract, cairo_path
+    )
```

Comment:
What would happen if compiled_contract is ContractClass?

Reply:
it cannot be `ContractClass` - it's an optional string right now. Is something wrong with that?

Reply:
<img width="461" alt="image" src="https://user-images.githubusercontent.com/61014013/177319466-1e7e5e16-fac5-4257-8e5a-ce95020f2112.png">

Are you sure? Because a few lines above it is Union[str, ContractClass].
Or am I missing something?